src/main/java/com/depromeet/domain/member/application/MemberService.java

```diff
@@ -184,6 +193,23 @@ public void updateFcmToken(UpdateFcmTokenRequest updateFcmTokenRequest) {
         currentMember.updateFcmToken(currentMember.getFcmInfo(), updateFcmTokenRequest.fcmToken());
     }
 
+    @Transactional(readOnly = true)
+    public void pushNotificationMissionRequest() {
+        LocalDateTime now = LocalDateTime.now();
+        List<Member> nonCompleteMissions = memberRepository.findNonCompleteMissions(now);
```

Comment:
findMissionNonCompletedMembers 는 어떤가용?

```suggestion
        List<Member> nonMissionNonCompletedMembers = memberRepository.findMissionNonCompletedMembers(now);
```